content/blog/2020/05/2020-05-18-read-only-jenkins-announcement.adoc

```diff
@@ -0,0 +1,126 @@
+---
+layout: post
+title: "Read-only Jenkins Configuration"
+tags:
+- system-read
+- extended-read
+- configuration-as-code
+- announcement
+author: timja
+---
+
+I'm excited to announce that the 'read-only' Jenkins feature is now available.
+
+You will want to use at least version TODO (waiting for agent to be released) of Jenkins.
+
+Read-only Jenkins currently allows users to have access to:
+
+* job configuration
+* system configuration
+* plugin manager
+* system logs
+* cloud configuration
+* agent configuration
+* agent logs
+
+For more planned integrations see the jira:JENKINS-12548[] epic.
+
+Read-only Jenkins is split into three permissions:
+
+* Job/ExtendedRead
+  - existed since 2009 but the UI didn't do anything to indicate to the users
+  that they couldn't edit the job configuration page, this has now been adapted to the new read-only engine.
+* Agent/ExtendedRead
+  - existed since 2013 but it was undocumented and only allowed access to API and no UI
+* Overall/SystemRead
```

Comment:
```suggestion
* **Overall/SystemRead** - System-wide read-only access.
     It is very useful for Jenkins instances managed as code, e.g. with help of the plugin:configuration-as-code[Jenkins Configuration as Code Plugin].
  - Introduced in Jenkins 2.222 as a part of jep:224[Readonly system configuration]
```

Reply:
Misleading description fixed up in https://github.com/jenkins-infra/jenkins.io/pull/3897